dev/Common/TelemetryHelper.h

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+#pragma once
+class TelemetryHelper
+{
+public:
+    TelemetryHelper()
```

Comment:
TelemetryHelper does not look like an appropriate name of this file/class here.
This class is helping in getting App information. AppInfo or AppProperties or similar name would be more appropriate.

Reply:
I wanted to write a general helper that could be used by others. This way, later more helpers for functionalities beyond just app information or properties could be added, which is why I kept it generic.